src/Factory/Api/Mastodon/Status.php

```diff
@@ -87,7 +87,7 @@ public function __construct(LoggerInterface $logger, Database $dba,
 	 * @throws HTTPException\InternalServerErrorException
 	 * @throws ImagickException|HTTPException\NotFoundException
 	 */
-	public function createFromUriId(int $uriId, int $uid = 0, bool $display_quote = false, bool $reblog = true, bool $in_reply_status = true): \Friendica\Object\Api\Mastodon\Status
+	public function createFromUriId(int $uriId, int $uid = 0, bool $display_quote, bool $reblog = true, bool $in_reply_status = true): \Friendica\Object\Api\Mastodon\Status
```

Comment:
If the `$uid` parameter is also mandatory, please remove the default value. Otherwise the `$display_quote` parameter needs to be moved another step left.

Reply:
Yeah, I had removed it to find all occurrences that I might have missed. I restored the functionality.